### ISSUE TITLE
Need Dictionary Object.

### ISSUE THREAD

**Issue by HAJ523** [rsek/datasworn#3]
With the move to arrays over objects Moves, Assets, Etc. must now be referenced by the index instead of by their name. This requires all users to potentially build their own indexes on the data.

There should be included indexes for each object so that there is simple access to translate from one to the other.

**Comment by rsek** [comment]
hmm. well, the (admittedly very limited) contexts i've worked in have never made this necessary... at least at this scale, where the largest array is ~80 objects or so? in JS i'd filter the relevant array by the `Name` key and call it a day, but i have this sneaking suspicion that any reasonable standard i'm a bit of a script kiddie.

so it might be important for reasons (performance, presumably?) that i don't know about! which doesn't put your request outside of scope for the _project_, necessarily. but definitely puts it outside of the scope of what i'm ready, willing, and able to maintain. it also makes me a _singularly bad_ candidate for producing the data you're seeking. a little like someone cooking while suffering from anosmia.

in other words: if you (or anyone else) want to submit a PR for it and maintain it (preferably as a discrete file), go for it. :)
